src/components/Match/VisionMap.jsx

```diff
@@ -20,7 +20,7 @@ import {
 const obsWard = (style, stroke, iconSize) => (<svg style={style} width={iconSize} height={iconSize} xmlns="http://www.w3.org/2000/svg">
   <g>
     <title>Observer</title>
-    <circle fill="#ffff00" strokeWidth="5" stroke={stroke} r={iconSize * 0.4} cy={iconSize / 2} cx={iconSize / 2} fillOpacity="0.4" />
+    <circle fill="#ffff00" strokeWidth="2.5" stroke={stroke} r={iconSize * 0.4} cy={iconSize / 2} cx={iconSize / 2} fillOpacity="0.3" />
```

Comment:
You have 2 for sentries. And since this is just the border, I think both of them should have the same value 2.